### PR TITLE
add neuron web add feature creating club

### PR DESCRIPTION
**Pull request by QingyangKong**
Add support for neuron web.
Add feature creating club.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,8 +2,20 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import routes from './routes'
 import './index.css'
+import Home from './containers/Home/index'
```
Comment (Keith-CY) [inline src/index.js]: `index` could be omitted, `import Home from './containers/Home'`

File: src/stores/newStore/index.js [hunk 2]
```diff
@@ -27,11 +31,64 @@ class NewStore {
...
+        .then(receipt => {
+          if (receipt.errorMessage === null) {
+            alert('社团创建成功', '优秀', [
+              { text : '是', onPress : this.handleOK },
+            ])
+          } else {
+            throw new Error(receipt.errroMessage)
+          }
+        })
+        .catch(err => {
```
Comment (Keith-CY) [inline src/stores/newStore/index.js]: unnecessary `catch`

**Comment by Keith-CY** [comment]
conflicts

File: src/index.js (hunk 1 above)
Comment (wangwenyue) [reply src/index.js]: This part can be delete. Not be used.

Comment (QingyangKong) [reply src/index.js]: yep, deleted!

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**